Assets/Scripts/Interactable/Booster.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Booster : RotationalInteractable
+{
+
+    #region Editors Fields
+
+    [SerializeField] private float _boosterSpeed = 1.0f;
+
+    #endregion
+
+    #region Private Variables
+
+    private Rigidbody _rb;
```

Comment:
Move the booster script to the `Driving Monitor` and have this variable as a `[SerializedField]`